crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -0,0 +1,234 @@
+function CrowdsourceHinter(runtime, element, data){
+    //switching units back to a previous question will make a second hinter start up.
+    //executeHinter is used to disable the hinter after switching units in an edX course.
+    var executeHinter = true;
+    $(".crowdsourcehinter_block", element).hide();
+
+    if(executeHinter){
+    var isShowingHintFeedback = false;
+    var voted = false;
+    $(".csh_HintsToUse", element).text("");
+
+    function stopScript(){
+    //This function is used to prevent a particular instance of the hinter from acting after
+    //switching between edX course's units. 
+        executeHinter = false;
+    }
+    Logger.listen('seq_next', null, stopScript);
+    Logger.listen('seq_prev', null, stopScript);
+    Logger.listen('seq_goto', null, stopScript);
```

Comment:
This is brittle. If I have a learning sequence below a problem+hinter, this will break. 

I would not fix this right now beyond a HACK/TODO comment.